CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Linkage Checker Enforcer Rule Change Log
 
+## 1.2.1
+* Linkage Checker properly handles invalid class files missing method body.
```

Comment:
Linkage Checker handles class files containing methods without a body.

(I'm not certain those are truly invalid. I'd need to check the JVM spec.)